python/examples/bend-flux.py

```diff
@@ -99,6 +99,6 @@
 plt.plot(wl,Ts,'ro-',label='transmittance')
 plt.plot(wl,1-Rs-Ts,'go-',label='loss')
 plt.axis([5.0, 10.0, 0, 1])
-plt.xlabel("wavelength (um)")
+plt.xlabel("wavelength (μm)")
```

Comment:
Unicode literals will break in Python 2. If you add this line to the top of the file:
```python
# -*- coding: utf-8 -*-
```
then it should work in python 2 and 3.

Reply:
Thanks for the suggestion. Fixed.